go/engine/identify2_with_uid.go

```diff
@@ -196,7 +196,7 @@ func (e *Identify2WithUID) runReturnError(ctx *Context) (err error) {
 		e.G().Log.Debug("- Released singleflight lock")
 	}()
 
-	if e.loadAssertion(); err != nil {
+	if err = e.loadAssertion(); err != nil {
```

Comment:
There are lints that catch this sort of thing. Does the `if` statement confuse them, or have we not turned them on yet?